task_3/writeFile.js

```diff
@@ -0,0 +1,34 @@
+const fsAsync = require('fs').promises;
```

Comment:
+1 for writing `csv` generator script. The file name (`writeFile`), though, might be a little bit confusing,